extension/entitystore/serviceprovider.go

```diff
@@ -39,8 +40,13 @@ const (
 	ServiceNameSourceUnknown           = "Unknown"
 	ServiceNameSourceUserConfiguration = "UserConfiguration"
 
-	jitterMax = 180
-	jitterMin = 60
+	describeTagsJitterMax = 3600
+	describeTagsJitterMin = 3000
```

Comment:
jitter between 50 to 60 minutes for successful runs so they don't all occur at the same time and causing burst throttling issue.